mkdocs.yml

```diff
@@ -80,7 +80,7 @@ nav:
       - "Search and access of restricted datasets": "tutorials/restricted-datasets.ipynb"
       - "Reproducing NASA sea level rise infographic": "tutorials/SSL.ipynb"
       - "Accessing and visualizing EMIT data with a few lines of code": "tutorials/emit-earthaccess.ipynb"
-      # - 'Distributing workloads with Dask clusters': 'tutorials/dask.ipynb'
+        # - 'Distributing workloads with Dask clusters': 'tutorials/dask.ipynb'
```

Comment:
This is so nitpicky, but I don't like this. When I comment a YAML line, the indentation of the comment is sometimes significant, as it was in this case. Can we turn this off?

Reply:
Nitpicky is fine by me :-)

Let me see if we can configure this.

However, if not, I suggest we actually remove the line. In this case, it's not a comment, per se, but rather something "commented out". To me, this is akin to committing commented out code, which I don't like. My take is that if you want to comment out something, just delete it. If you want to know what it was, look at the commit history, right?

Reply:
Thanks for looking in to that!

Agree totally on commented-out code. But I also don't know who added that comment and what they intended (sorry if it feels like we're dogpiling on whoever wrote it: IF we're going to comment code, I like a reason to be also included, like `# we're waiting on issue #xyz to be resolved to re-add this tutorial:`), so I chose to set that aspect aside :)

Reply:
Doesn't look like I can control the indentation of that commented-out line. However, the indentation is changed _only_ when it is the last item in the list. If I move the item to any other position within that list, it is left as-is.

I might open an issue against yamlfmt about this behavior.

I the meantime, which would you prefer: (a) remove the line in question, or (b) move the line to a different position in the list (first, next-to-last, other)?

My preference would be to remove it, but I'm not vehemently opposed to simply changing its position.

Reply:
>I might open an issue against yamlfmt about this behavior.

Seems like the right next step, can't imagine why that behavior would be intended!

>I the meantime, which would you prefer: (a) remove the line in question, or (b) move the line to a different position in the list (first, next-to-last, other)?
>
>My preference would be to remove it, but I'm not vehemently opposed to simply changing its position.

Up to you, but I think if we remove the comment we should create a ticket to "figure out what to do with the dask tutorial"; re-add it to the nav (what needs to be done before we can do that?), or remove the tutorial from the repo?

Reply:
Actually, we should have a ticket for that either way. The comment is not doing the same job as a ticket :laughing: I'll open one.

Reply:
https://github.com/nsidc/earthaccess/issues/569

My vote is now for "delete", we'll figure it out through the issue :)

Reply:
Ok, I'm going to remove the comment. By the way, this indentation behavior is already known: https://github.com/google/yamlfmt/issues/112

I see that @betolink added the commented out line in this commit: https://github.com/nsidc/earthaccess/commit/dab9da4b290424dc4de2bd2212a8d1212019b947#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2R66

It looks like it never existed _uncommented_ and it was originally added as a commented-out line, perhaps as a reminder. Since this comment doesn't seem to have done its job as a reminder (as you mentioned), your opening a new ticket sounds like a better reminder :-)

Reply:
:laughing: We did the exact same sleuthing in different threads: https://github.com/nsidc/earthaccess/issues/569#issuecomment-2106313821